source/src/main/java/org/cerberus/core/service/csvfile/ICsvFileService.java

```diff
@@ -36,7 +36,18 @@ public interface ICsvFileService {
      * @param separator
      * @param columnsToGet
      * @param columnsToHide
+	 * @param ignoreNoMatchColumns      if <code>true</code> then populate all
+	 *                                  non-matched column with {@link String} empty
+	 *                                  value (""). Otherwise, all non-matched
+	 *                                  columns will not be included in the answer.
+	 *                                  Use {@link #IGNORE_NON_MATCHED_COLUMNS} &
+	 *                                  {@link #DO_NOT_IGNORE_NON_MATCHED_COLUMNS} for
+	 *                                  better readability
+	 * @param defaultNoMatchColumnValue the default value to set to any non-matched
+	 *                                  column if necessary
      * @return
      */
-    AnswerList<HashMap<String, String>> parseCSVFile(String urlToCSVFile, String separator, HashMap<String, String> columnsToGet,List<String> columnsToHide, TestCaseExecution execution);
+    AnswerList<HashMap<String, String>> parseCSVFile(String urlToCSVFile, String separator, HashMap<String, String> columnsToGet,List<String> columnsToHide, boolean ignoreNoMatchColumns, String defaultNoMatchColumnValue, TestCaseExecution execution);
+    boolean IGNORE_NON_MATCHED_COLUMNS = true;
```

Comment:
Can we avoid creating those and use boolean values directly ?

Reply:
These variables were used for better readability. The latest commit (#17ba4e0) removes them.